src/screens/Ticketing/Purchase/Overview/components/DurationSelection.tsx

```diff
@@ -0,0 +1,129 @@
+import React from 'react';
+import {DurationSelectionTexts, useTranslation} from '@atb/translations';
+import {
+  getReferenceDataName,
+  productIsSellableInApp,
+} from '@atb/reference-data/utils';
+import ThemeText from '@atb/components/text';
+import {InteractiveColor} from '@atb/theme/colors';
+import {TouchableOpacity, View} from 'react-native';
+import {StyleSheet, useTheme} from '@atb/theme';
+import {ScrollView} from 'react-native';
+import {PreassignedFareProduct} from '@atb/reference-data/types';
+import {useFirestoreConfiguration} from '@atb/configuration/FirestoreConfigurationContext';
+
+type DurationSelectionProps = {
+  color: InteractiveColor;
+  selectedProduct: PreassignedFareProduct;
+  setSelectedProduct: (product: PreassignedFareProduct) => void;
+};
+
+export default function DurationSelection({
+  color,
+  selectedProduct,
+  setSelectedProduct,
+}: DurationSelectionProps) {
+  const {t, language} = useTranslation();
+  const styles = useStyles();
+
+  const {preassignedFareproducts} = useFirestoreConfiguration();
+
+  const selectableProducts = preassignedFareproducts
+    .filter(productIsSellableInApp)
+    .filter((p) => p.type === selectedProduct.type);
+
+  return (
+    <View style={styles.container}>
+      <ThemeText type="body__secondary" color="secondary">
+        {t(DurationSelectionTexts.title)}
+      </ThemeText>
+      <ScrollView
+        style={styles.durationScrollView}
+        contentContainerStyle={styles.durationContentContainer}
+        horizontal={true}
+        showsHorizontalScrollIndicator={false}
+      >
+        {selectableProducts.map((fp, i) => (
+          <DurationChip
+            color={color}
+            text={getReferenceDataName(fp, language)}
```

Comment:
In the Figma-sketches the product duration is used, not product name (e.g. "30 dager", not "30-dagersbillett"). The field `durationDays` is added to the `PreassignedFareProduct` in Firestore in staging to enable this. I don't know if there has been any office discussions changing this, but if not the implementation should be changed to be aligned with how it is in Figma and described in the issue.

Reply:
👍